Firestore/Example/Benchmarks/remote_document_cache_benchmark.mm

```diff
@@ -15,7 +15,6 @@
  */
 
 #import <FirebaseFirestore/FirebaseFirestore.h>
-#import "FirebaseCore/Internal/FirebaseCoreInternal.h"
```

Comment:
Should we still import `FirebaseApp.h` at least? Seems like we're relying on an implicit dependency now.

Reply:
Good catch!